modules/core/24-host/channel_keys.go

```diff
@@ -52,3 +54,8 @@ func ChannelRestoreKey(portID, channelID string) []byte {
 func channelPath(portID, channelID string) string {
 	return fmt.Sprintf("%s/%s/%s/%s", KeyPortPrefix, portID, KeyChannelPrefix, channelID)
 }
+
+// ErrorPath stores the ErrorReceipt in the case that a chain does not accept the proposed upgrade
+func ErrorPath(portID, channelID string) string {
+	return fmt.Sprintf("%s/%s/%s/%s/%s/%s", KeyChannelUpgradePrefix, KeyPortPrefix, portID, KeyChannelPrefix, channelID, KeyUpgradeError)
+}
```

Comment:
do we need to also add an `ErrorKey`, or maybe a more explicit `ChannelUpgradeErrorKey`

Reply:
I think @chatton is referring to the key function which returns the path as a `[]byte`

Reply:
I was referring to having a `ErrorPath -> string` and `ErrorKey -> []byte`

Reply:
got it, that makes sense now 🙈 . i'll add that